src/ingest-pipeline/airflow/dags/multiome.py

```diff
@@ -0,0 +1,348 @@
+                cwl_workflows[2],
+                "--input_dir",
+                # This pipeline invocation runs in a 'hubmap_ui' subdirectory,
+                # so use the parent directory as input
+                "..",
+            ]
+
+            return join_quote_command_str(command)
+
+        t_build_cmd1 = PythonOperator(
+            task_id="build_cmd1",
+            python_callable=build_cwltool_cmd1,
+            provide_context=True,
+        )
+
+        t_build_cmd2 = PythonOperator(
+            task_id="build_cmd2",
+            python_callable=build_cwltool_cmd2,
+            provide_context=True,
+        )
+
+        t_build_cmd3 = PythonOperator(
+            task_id="build_cmd3",
+            python_callable=build_cwltool_cmd3,
+            provide_context=True,
+        )
+
+        t_pipeline_exec = BashOperator(
+            task_id="pipeline_exec",
+            bash_command=""" \
+            tmp_dir={{tmp_dir_path(run_id)}} ; \
+            {{ti.xcom_pull(task_ids='build_cmd1')}} > $tmp_dir/session.log 2>&1 ; \
+            echo $?
+            """,
+        )
+
+        t_pipeline_exec_azimuth_annotate = BashOperator(
+            task_id="pipeline_exec_azimuth_annotate",
+            bash_command=""" \
+            tmp_dir={{tmp_dir_path(run_id)}} ; \
+            cd "$tmp_dir"/cwl_out ; \
+            {{ti.xcom_pull(task_ids='build_cmd2')}} >> $tmp_dir/session.log 2>&1 ; \
+            echo $?
+            """,
+        )
+
+        t_convert_for_ui = BashOperator(
+            task_id="convert_for_ui",
+            bash_command=""" \
+            tmp_dir={{tmp_dir_path(run_id)}} ; \
+            ds_dir="{{ti.xcom_pull(task_ids="send_create_dataset")}}" ; \
+            cd "$tmp_dir"/cwl_out ; \
+            mkdir -p hubmap_ui ; \
+            cd hubmap_ui ; \
+            {{ti.xcom_pull(task_ids='build_cmd3')}} >> $tmp_dir/session.log 2>&1 ; \
+            echo $?
+            """,
+        )
+
+        t_maybe_keep_cwl1 = BranchPythonOperator(
+            task_id="maybe_keep_cwl1",
+            python_callable=utils.pythonop_maybe_keep,
+            provide_context=True,
+            op_kwargs={
+                "next_op": "prepare_cwl2",
+                "bail_op": "set_dataset_error",
+                "test_op": "pipeline_exec",
+            },
+        )
+
+        t_maybe_keep_cwl2 = BranchPythonOperator(
+            task_id="maybe_keep_cwl2",
+            python_callable=utils.pythonop_maybe_keep,
+            provide_context=True,
+            op_kwargs={
+                "next_op": "prepare_cwl3",
+                "bail_op": "set_dataset_error",
+                "test_op": "pipeline_exec_azimuth_annotate",
+            },
+        )
+
+        t_maybe_keep_cwl3 = BranchPythonOperator(
+            task_id="maybe_keep_cwl3",
+            python_callable=utils.pythonop_maybe_keep,
+            provide_context=True,
+            op_kwargs={
+                "next_op": "prepare_cwl4",
```

Comment:
The next_op doesn't exists, maybe it should be move_data (next step in the declared flow)